app/src/main/java/com/yeonae/chamelezone/view/home/presenter/HomeContract.kt

```diff
@@ -1,19 +1,24 @@
 package com.yeonae.chamelezone.view.home.presenter
 
+import com.yeonae.chamelezone.data.model.LikeStatusItem
 import com.yeonae.chamelezone.network.model.PlaceResponse
 import com.yeonae.chamelezone.network.room.entity.UserEntity
 
 interface HomeContract {
 
     interface View {
         var presenter: Presenter
-        fun showHomeList(response: List<PlaceResponse>)
+        fun showHomeList(place: List<PlaceResponse>)
```

Comment:
```suggestion
        fun showHomeList(placeList: List<PlaceResponse>)
```
복수개를 의미할 수 있는 네이밍으로 변경이 필요합니다.

Reply:
수정했습니다
0d15d0d